google-cloud-storage/src/main/java/org/jclouds/googlecloudstorage/features/ObjectApi.java

```diff
@@ -123,7 +124,7 @@
    @Fallback(NullOnNotFoundOr404.class)
    @Nullable
    GoogleCloudStorageObject getObject(@PathParam("bucket") String bucketName, @PathParam("object") String objectName,
-            GetObjectOptions options);
+         HttpRequestOptions options);
```

Comment:
Sorry I misunderstood this -- I looked at `org.jclouds.blobstore.options.GetOptions` and not `org.jclouds.http.options.GetOptions` which implements `HttpRequestOptions`.  We can pass `GetObjectOptions` as you suggest.